pyproject.toml

```diff
@@ -1,4 +1,51 @@
+[tool.poetry]
+name = "pyopia"
+version = "0"
+description = "A Python Ocean Particle Image Analysis toolbox."
+authors = ["Emlyn Davies <emlyn.davies@sintef.no>"]
+readme = "README.md"
+packages = [{include = "pyopia"}]
+
+[tool.poetry.dependencies]
```

Comment:
Are all of these packages direct installs, or are some of them dependencies of primary packages? Poetry takes care of pinning dependencies in the lock file, so that you can place only the human readable ones here. (Read more: 

Also, it is possible to organize dependencies by groups, to be more explicit about in what setting a package is used: https://python-poetry.org/docs/managing-dependencies/

For example, I imagine there's one group for testing, another for user running of notebooks, and a third for building documentation. However, splitting into groups is only for convenience (faster builds if installing requirements often in pipelines, for example) and user-friendliness, so if it ain't broken...

Reply:
there is probably a mixture (this was more-or-less created from the old requirements.txt). i can go through and check which ones are not primary packages

Reply:
this is probably the most elegant solution to #14